src/snowflake/snowpark/types.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, length: Optional[int] = None) -> None:
         self.length = length
 
     def __repr__(self) -> str:
-        if self.length:
+        if self.length and self.length != self._MAX_LENGTH:
```

Comment:
Why do we display the length when it's `MAX_LENGTH`?

Reply:
The code is when the length is not max_length.